src/utils/auto-box-collider.js

```diff
@@ -7,8 +24,9 @@ export function getBox(entity, boxRoot) {
 
   entity.object3D.updateMatrices(true, true);
   boxRoot.updateMatrices(true, true);
+  boxRoot.updateMatrixWorld(true);
 
-  box.setFromObject(boxRoot);
+  computeObjectAABB(boxRoot, box);
```

Comment:
Would like it if this function name indicated that the box that is returned is not the smallest AABB that contains the object.
I know I don't like `computeObjectAABBButNotNecessarilyTheSmallestOneThatContainsTheObject` but I haven't thought of another one. 

Perhaps a comment instead of a rename would help, since we all guessed wrong about whether this algorithm would return the smallest AABB.